<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgotMyPassword</class>
 <widget class="QDialog" name="ForgotMyPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ForgotMyPassword {border-image: url(:/images/images/back.jpg) 0 0 0 0 
;}</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>4</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="user">
        <property name="styleSheet">
         <string notr="true">#user{color:white;}</string>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="userL">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#userL{border-image:url(:/images/images/gameground.png);padding-left:10px;}</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="cursorPosition">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="email">
        <property name="styleSheet">
         <string notr="true">#email{color:white;}</string>
        </property>
        <property name="text">
         <string>email address</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="emailL">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#emailL{border-image:
url(:/images/images/gameground.png);
padding-left:10px;

}</string>
        </property>
        <property name="cursorPosition">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="check">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#check{background-color:green;}
</string>
   </property>
   <property name="text">
    <string>check</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>241</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="pass">
            <property name="styleSheet">
             <string notr="true">#pass{color:white;}</string>
            </property>
            <property name="text">
             <string>new password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="passL">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">#passL{border-image:
url(:/images/images/gameground.png);
padding-left:10px;

}</string>
            </property>
            <property name="cursorPosition">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="confirm_pass">
            <property name="styleSheet">
             <string notr="true">#confirm_pass{color:white;}</string>
            </property>
            <property name="text">
             <string>confirm password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="confirm_passL">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">#confirm_passL{border-image:
url(:/images/images/gameground.png);
padding-left:10px;

}</string>
            </property>
            <property name="cursorPosition">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="confirm">
        <property name="styleSheet">
         <string notr="true">#confirm{background-color:green;opacity: 0.5;;}
</string>
        </property>
        <property name="text">
         <string>confirm</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
